run_iwyu_tests.py

```diff
@@ -127,6 +133,15 @@ def RunTestFile(cc_file):
     # https://mesonbuild.com/Unit-tests.html#skipped-tests-and-hard-errors
     print('Skipped %s: %s' % (cc_file, e))
     return 77
+  except AssertionError as e:
+    if xfail:
+      print("%s: Expected failure" % cc_file)
```

Comment:
Ah, sorry, one more thing: historically, our Python has preferred single-quotes for strings unless the string value contains single quotes. So let's use singles here and below.

Otherwise everything LGTM, and I've tested it locally to great effect, so if you would also squash the branch to a single commit, that'd be great.

Reply:
Done. Squashed and force pushed.